common/src/float_ops.rs

```diff
@@ -188,14 +188,17 @@ pub fn format_general(
     magnitude: f64,
     case: Case,
     alternate_form: bool,
+    is_format_type_none: Option<bool>,
```

Comment:
by concept, this function doesn't recognize about format type. adding a parameter using this name doesn't look well fit and breaking abstraction layer.
And `Some(false)` and `None` doesn't have difference for this function.

```suggestion
    always_shows_fract: bool,
```